forge-core/src/main/java/forge/card/CardRules.java

```diff
@@ -546,9 +563,7 @@ public final void parseLine(final String line) {
                     } else if ("DeckHas".equals(key)) {
                         has = new DeckHints(value);
                     } else if ("Defense".equals(key)) {
-                        this.faces[this.curFace].setDefense(value);
-                    } else if ("Draft".equals(key)) {
-                        this.faces[this.curFace].addDraftAction(value);
```

Comment:
bad rebase?

Reply:
Didn't realize at 3 AM that the IDE's merge tool was showing me the whole file and not just a single change. Fixed now.